lib/Bling.js

```diff
@@ -317,7 +317,7 @@ var Bling = (_temp2 = _class = function (_Component) {
                 slotSize = sizeMapping[0] && sizeMapping[0].slot;
             }
 
-            return slotSize;
+            return JSON.stringify(slotSize) === "[0,0]" ? [] : slotSize;
```

Comment:
@evheniy Can you please add warning log here and in the other file? 
We should prevent developers to write wrong code, since this fix doesn't solve the issue, just 'patch' it and it is 'temporary' (if we switch to use gpt.js directly, without this library, we will again have the same issue, so developers should fix the incorrectly written size in the siteapp, whenever the warning is there (later, when we have no warnings, we might even remove this 'patch')

Thanks

Reply:
@majda 
This library works only on client side so I don't think that adding console.warn is a good idea as our clients can see our warnings. 
Another case is old browser - if it does not have console object it can break js.